### PR TITLE
248/invest page investing section

### PR DESCRIPTION
**Pull request by NewtonOmbese**
# Description

Created a section that describes the Invest page investing section that shows an image on the left side, a paragraph of text and a title. This component displays provided text and images side by side on the Invest page investing section using the reusable text and image banner in issue https://github.com/italanta/elewa-group/issues/31 and the reusable animated button https://github.com/italanta/elewa-group/issues/11. 

To achieve this, I needed to:
- Step 1: Create the Elewa Invest page investing section component
```
nx g @nrwl/angular:component pages/elewa/invest/elewa-invest-investing-in-elewa-section
```
- Make an import of the banner module to be reused in the component
- Make an import of the button module to be reused in the banner component

Fixes # (issue  https://github.com/italanta/elewa-group/issues/248) 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Desktop View
invest3-desktop

Mobile View
invest3-mobile

# How Has This Been Tested?
Step 1: Added the component to invest-page-component.html
Step 2: Started the server by running ```nx serve elewa-group-website```
Step 3: Using chrome developer tools to test on the responsiveness of the website

# Checklist:t
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by NewtonOmbese** [comment]
Review and Give feedback @IanOdhiambo9

File: libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
         <div id="text-content" class="content">
             <h1 class="title">{{titleText}}</h1>
             <p class="paragraph" *ngFor="let paragraph of paragraphTexts">{{paragraph}}</p>
+            <div *ngIf="!showButton">
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html:6]: should be *ngIf="showButton"

File: libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts [hunk 2]
```diff
@@ -11,4 +11,7 @@ export class ElewaGroupImageAndTextBannerComponent {
   @Input() titleText = '';
   @Input() imagePlacement = 'right';
   @Input() backgroundColor = 'grey';
+  buttonText = "Join the waiting list";
+  url = "home/en";
+  showButton = false;
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts:16]: all these should be inputs and optional

File: libs/pages/elewa/invest/src/lib/components/elewa-invest-investing-in-elewa-section/elewa-invest-investing-in-elewa-section.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<elewa-group-elewa-group-image-and-text-banner [imageURL]="image" [titleText]="titleText" [backgroundColor]="backgroundColor" [paragraphTexts]="paragraph" [imagePlacement]="imagePlacement"></elewa-group-elewa-group-image-and-text-banner>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/invest/src/lib/components/elewa-invest-investing-in-elewa-section/elewa-invest-investing-in-elewa-section.component.html:1]: button options should be added here as well.

File: libs/pages/elewa/invest/src/lib/pages-elewa-invest.module.ts [hunk 4]
```diff
@@ -1,17 +1,18 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-
+import { BannersModule } from '@elewa-group/features/components/banners';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/invest/src/lib/pages-elewa-invest.module.ts:3]: should be below elements

File: libs/pages/elewa/invest/src/lib/pages/invest-page/invest-page.component.html [hunk 5]
```diff
@@ -1,2 +1,3 @@
 <elewa-group-elewa-invest-hero></elewa-group-elewa-invest-hero>
+<elewa-group-elewa-invest-investing-in-elewa-section></elewa-group-elewa-invest-investing-in-elewa-section>
 <elewa-group-elewa-invest-detail-section></elewa-group-elewa-invest-detail-section>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/invest/src/lib/pages/invest-page/invest-page.component.html:3]: sync and resolve

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
sync and resolve conflicts

**Comment by NewtonOmbese** [comment]
@IanOdhiambo9  needed :wrench: fixes have been made review :memo: and give feedback :bulb:

File: libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html (hunk 1 above)
Comment (NewtonOmbese) [reply libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.html]: Resolved :100:

File: libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts (hunk 2 above)
Comment (NewtonOmbese) [reply libs/features/components/banners/src/lib/banners/elewa-group-image-and-text-banner/elewa-group-image-and-text-banner.component.ts]: Resolved :100:

File: libs/pages/elewa/invest/src/lib/components/elewa-invest-investing-in-elewa-section/elewa-invest-investing-in-elewa-section.component.html (hunk 3 above)
Comment (NewtonOmbese) [reply libs/pages/elewa/invest/src/lib/components/elewa-invest-investing-in-elewa-section/elewa-invest-investing-in-elewa-section.component.html]: Done :100:

File: libs/pages/elewa/invest/src/lib/pages-elewa-invest.module.ts (hunk 4 above)
Comment (NewtonOmbese) [reply libs/pages/elewa/invest/src/lib/pages-elewa-invest.module.ts]: fixed

File: libs/pages/elewa/invest/src/lib/pages/invest-page/invest-page.component.html (hunk 5 above)
Comment (NewtonOmbese) [reply libs/pages/elewa/invest/src/lib/pages/invest-page/invest-page.component.html]: Done

**Comment by NewtonOmbese** [comment]
@IanOdhiambo9  still awaiting for your feedback

**Comment by IanOdhiambo9** [comment]
The component is not on the page, check the link.

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**